.github/workflows/integration-tests.yml

```diff
@@ -85,7 +85,7 @@ jobs:
         with:
           python-version: "3.12"
       - run: git clone https://github.com/pyvista/pyvistaqt.git --single-branch
-      - uses: pyvista/setup-headless-display-action@main
+      - uses: pyvista/setup-headless-display-action@7d84ae825e6d9297a8e99bdbbae20d1b919a0b19
```

Comment:
Were these GHAs intentionally referencing `main` for bleeding edge and unreleased commits?